Library/Formula/gringo.rb

```diff
@@ -3,16 +3,16 @@
 class Gringo < Formula
   desc "Grounder to translate user-provided logic programs"
   homepage 'http://potassco.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/potassco/gringo/4.4.0/gringo-4.4.0-source.tar.gz'
-  sha1 'c39a1c3cfe64b62e39e6abcc8f813e2d1d17251e'
+  url 'https://downloads.sourceforge.net/project/potassco/gringo/4.5.0/gringo-4.5.0-source.tar.gz'
+  sha1 'bbe67ed1d5c6f5ca6d5958e04b706595d39b4653'
```

Comment:
Same comments about `sha256` and not editing the `bottle do` block apply here as well.